geode-core/src/main/java/org/apache/geode/management/internal/api/LocatorClusterManagementService.java

```diff
@@ -155,23 +159,24 @@ public ClusterManagementResult create(CacheElement config) {
 
     // add the config object which includes the HATOS information of the element created
     if (result.isSuccessful()) {
-      result.setResult(Collections.singletonList(config));
+      return get(config);
```

Comment:
also note: because response type is no longer required to extend the config type, `create` now invokes `list` to query back the newly-created element, rather than just returning the original create request.  This required minor changes to a few tests that heavily mocked the create flow.

Reply:
reverted this change